.project

```diff
@@ -14,4 +14,15 @@
 	<natures>
 		<nature>org.eclipse.m2e.core.maven2Nature</nature>
 	</natures>
+	<filteredResources>
```

Comment:
There have been quite a few PRs with this all in it. Can you separate it into it's own PR that we can merge so the other PRs don't have this in it (that doesn't pertain to the changes you're making)?